source/slang/slang-emit-spirv.cpp

```diff
@@ -2724,6 +2758,13 @@ struct SPIRVEmitContext
                 const auto memorySemantics =  emitIntConstant(IRIntegerValue{SpvMemorySemanticsMaskNone}, builder.getUIntType());
                 return emitOpAtomicIDecrement(parent, inst, inst->getFullType(), inst->getOperand(0), memoryScope, memorySemantics);
             }
+        case kIROp_GetLegalizedSPIRVGlobalParamAddr:
```

Comment:
Is this needed? My assumption is that this inst should be legalized out during spirv legalization.

Reply:
`processGlobalParam` replaces all `globalParam` with a `load(globalParam)`, it walks the use's list.

adding a intrinsic meant the intrinsic will get a `load(globalParam)` after legalization from a `globalParam`.

Reply:
Yeah, I think you still need that logic in `insertLoadAtLatestLocation` to handle the case where a user is `GetLegalizedParamAddr`, in which case you replace that getAddr inst with the pointer itself and remove the inst.

Reply:
I understand what you are saying, will do 👍